### ISSUE TITLE
Python API fails to download data on MOLGENIS 7.2.4 with a 500 server error

### ISSUE THREAD

**Issue by djvanenckevort** [molgenis/molgenis#8080]
### How to Reproduce
Clone the repo at 
https://github.com/djvanenckevort/molgenis-app-bbmri-eric-scripts and perform the instructions in the README.

Edit the script and change num=0 to num=2000 in the  data-check.py to adjust for num=0 no longer being supported in MOLGENIS 7 (lines 160, 169, 179 and 188)

Run the script.

### Expected behavior
The script runs and creates a report.

### Observed behavior
The script fails with a 500 error on the call to load the biobank data.

**Comment by fdlk** [comment]
@djvanenckevort Do you have details on the REST call that fails and/or a stacktrace?

**Comment by djvanenckevort** [comment]
https://directory.bbmri-eric.eu/api/v1/eu_bbmri_eric_biobanks?expand=contact&expand=collections&expand=country&num=2000&start=0

Internal Server Error
The server encountered an internal error or misconfiguration and was unable to complete your request.

Please contact the server administrator, admin@molgenis.org and inform them of the time the error occurred, and anything you might have done that may have caused the error.

More information about this error may be available in the server error log.

Apache/2.2.15 (CentOS) Server at directory.bbmri-eric.eu Port 443

**Comment by dennishendriksen** [comment]
@djvanenckevort the page size (2000) results in a large amount of data to be generated due to the expanded attributes.
```
GET https://directory.bbmri-eric.eu/api/v1/eu_bbmri_eric_biobanks?num=2000&start=0
```
or
```
GET https://directory.bbmri-eric.eu/api/v1/eu_bbmri_eric_biobanks?expand=contact&expand=collections&expand=country&num=100&start=0
```
return results in a timely manner.

**Comment by dennishendriksen** [comment]
@djvanenckevort did my previous comment resolve your issue?

**Comment by djvanenckevort** [comment]
Yes, reducing the batch size does make the issue go away, but this is a regression in the performance between 5.2 and 7. In 5.2 the same script could download the same data in one go.
